lib/request-parser.js

```diff
@@ -7,10 +7,75 @@ const bodyParser = require("body-parser");
 const cookieParser = require("cookie-parser");
 const multer = require("multer");
 const tmp = require("tmp");
+const path = require("path");
+const util = require("./helpers/util");
 
 // Clean-up the temp directory, even if the app crashes
 tmp.setGracefulCleanup();
 
+/**
+ * Generates a middleware that parses multipart/form-data
+ */
+function generateMultipartFormDataMiddleware (options) {
+  options.allowAll = options.allowAll === undefined ? true : options.allowAll;
+  const uploader = multer(options);
```

Comment:
Traditionally one would pass in the `inMemory` flag; thoughts on making that compatible here? Otherwise folks will need to pull in multer and pass in a storage engine for in-memory. Which is fine, though it seems like a leaky abstraction point possibly?

Reply:
Can't we just check for the `inMemory` flag, and if it's set, then we create a storage engine automatically?

Reply:
Yea that was my question basically - if we wanted to do that 🙂 I'll add that.

Reply:
Added [here](https://github.com/APIDevTools/swagger-express-middleware/pull/165/files#diff-0819ff236dc445648af37b543cd2b958R22-R24)